### PR TITLE
fix: do not set mode name into variables

### PR DESCRIPTION
**Pull request by helgastogova**
see that https://github.com/kiveai/kive/pull/3141#issuecomment-2392997630

### CODE REVIEW

File: src/utilities/processAliasModes.ts [hunk 1]
```diff
@@ -2,28 +2,25 @@ const processAliasModes = (variables) => {
...
+    delete variable.aliasMode;
+    delete variable.aliasCollectionName;
 
     collector.push({
       ...variable,
-      values: variable.values.replace(
-        `{${aliasCollectionName}.`,
-        `{${aliasCollectionName}.${aliasMode.name}.`
-      )
-    })
```
Comment (helgastogova) [inline src/utilities/processAliasModes.ts:23]: changes are only here

**Comment by Alexloof** [comment]
Do you wanna commit the `dist` folder?

**Comment by helgastogova** [comment]
> Do you wanna commit the `dist` folder?

I was thinking about the fact if we commit it, we could just download the repo and import a plugin w/out running scripts. The folder will be "ready to import into figma"

What do you think?

**Comment by helgastogova** [comment]
> Do you wanna commit the `dist` folder?

We already have them committed in the original repo and I think it is nice that you can just download a zip folder with repo and import a plugin into Figma w/out running any scripts, what do you think?

**Comment by Alexloof** [comment]
> > Do you wanna commit the `dist` folder?
> 
> We already have them committed in the original repo and I think it is nice that you can just download a zip folder with repo and import a plugin into Figma w/out running any scripts, what do you think?

ok 👍

**Review by Alexloof — APPROVED** [review]

**Pull request merged**